### ISSUE TITLE
Either remove or update out of date samples

### ISSUE THREAD

**Issue by kjeremy** [MvvmCross/MvvmCross-Samples#122]
See title

**Comment by martijn00** [comment]
We need some help updating them. Anyone is free to open a PR to update a sample.

**Comment by acuntex** [comment]
> We need some help updating them. Anyone is free to open a PR to update a sample.

It's hard to update if the documentation is so thin, samples are not available and the hint to look at the playground in the documentation is also unnecessary if there are no specific samples for exactly this documentation. (e.g. WPF View Presenter)

I'm sure the devs know how the library works and probably also people who used it for a long time and upgraded the versions, but if you start with version 6, it's quite hard to start with the current documentation (whether it's textual or code).

**Comment by saamerm** [comment]
@kjeremy I don't think it makes sense to update or remove. If anyone is looking for the up to date samples, they could just check the last update date against the sample folder to know which ones to try.
<img width="701" alt="image" src="https://user-images.githubusercontent.com/8262287/82548141-d606fa00-9b28-11ea-9bc4-2703c1b33291.png">

There's a lot of work that has gone into each one of them, and sometimes no prior knowledge and just 30-60 minutes of research/stackoverflow work is all you need to do to fix them.
